test/e2e/multihoming_utils.go

```diff
@@ -354,6 +354,88 @@ func multiNetEgressLimitingIPBlockPolicy(
 	}
 }
 
+func multiNetIngressDenyAllPolicy(
+	policyName string,
+	policyFor string,
+	appliesFor metav1.LabelSelector,
+) *mnpapi.MultiNetworkPolicy {
+	return &mnpapi.MultiNetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: policyName,
+			Annotations: map[string]string{
+				PolicyForAnnotation: policyFor,
+			},
+		},
+		Spec: mnpapi.MultiNetworkPolicySpec{
+			PodSelector: appliesFor,
+			PolicyTypes: []mnpapi.MultiPolicyType{mnpapi.PolicyTypeIngress},
+		},
+	}
+}
+
+func multiNetIngressDenyAllEgressAllowAllPolicy(
+	policyName string,
+	policyFor string,
+	appliesFor metav1.LabelSelector,
+) *mnpapi.MultiNetworkPolicy {
+	return &mnpapi.MultiNetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: policyName,
+			Annotations: map[string]string{
+				PolicyForAnnotation: policyFor,
+			},
+		},
+		Spec: mnpapi.MultiNetworkPolicySpec{
+			PodSelector: appliesFor,
+			PolicyTypes: []mnpapi.MultiPolicyType{mnpapi.PolicyTypeIngress},
+			Egress: []mnpapi.MultiNetworkPolicyEgressRule{
+				mnpapi.MultiNetworkPolicyEgressRule{},
+			},
+		},
+	}
+}
+
+func multiNetEgressAllowAllPolicy(
+	policyName string,
+	policyFor string,
+	appliesFor metav1.LabelSelector,
+) *mnpapi.MultiNetworkPolicy {
+	return &mnpapi.MultiNetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: policyName,
+			Annotations: map[string]string{
+				PolicyForAnnotation: policyFor,
+			},
+		},
+		Spec: mnpapi.MultiNetworkPolicySpec{
+			PodSelector: appliesFor,
+			PolicyTypes: []mnpapi.MultiPolicyType{mnpapi.PolicyTypeEgress},
+			Egress: []mnpapi.MultiNetworkPolicyEgressRule{
+				mnpapi.MultiNetworkPolicyEgressRule{},
+			},
+		},
+	}
+}
+
+func multiNetEgressDenyAllPolicy(
+	policyName string,
+	policyFor string,
+	appliesFor metav1.LabelSelector,
+) *mnpapi.MultiNetworkPolicy {
+	return &mnpapi.MultiNetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: policyName,
+			Annotations: map[string]string{
+				PolicyForAnnotation: policyFor,
+			},
+		},
+		Spec: mnpapi.MultiNetworkPolicySpec{
+			PodSelector: appliesFor,
+			PolicyTypes: []mnpapi.MultiPolicyType{mnpapi.PolicyTypeEgress},
+		},
+	}
```

Comment:
nit: in all the 4 utils you added only the spec is changing? would you have a base function which is then edited in each sub function so that duplication can be avoided?
wdyt @maiqueb ofc I won't push for this, outside of this lgtm

Reply:
it for sure reduce code, question is, isnt it nicer to see as a whole the policy when it has a dedicated function ?
anyhow whatever you decide of course

Reply:
right what Miguel said ^
I'd make the utils layered and resuable => I don't think that destroys readability as long as the names are self-descriptive; otherwise this will bloat up soon

Reply:
example: https://github.com/ovn-org/ovn-kubernetes/blob/23c8e0cd55bd04b4f4711c789f1e556d55031c47/go-controller/pkg/ovn/policy_test.go#L47 which is based called by https://github.com/ovn-org/ovn-kubernetes/blob/23c8e0cd55bd04b4f4711c789f1e556d55031c47/go-controller/pkg/ovn/policy_test.go#L548 and https://github.com/ovn-org/ovn-kubernetes/blob/23c8e0cd55bd04b4f4711c789f1e556d55031c47/go-controller/pkg/ovn/policy_test.go#L513 imagine in future you will need to do various flavours of this,

Reply:
haha let's go with for now and as you add more tests you can figure out which utils are being frequently re-used to add further ones.